src/jit/utils.h

```diff
@@ -381,6 +381,15 @@ class PhasedVar
         return m_value;
     }
 
+    // Mark the value as read only
+    void MarkAsReadOnly() const
```

Comment:
Let me suggest:
``
    // Mark the value as read only; explicitly change the variable to the "read" phase.
``